dependents/source_repo.yml

```diff
@@ -26,6 +26,12 @@ jobs:
   dev:
     name: Development
     runs-on: ubuntu-20.04
+
+    # Sets permissions of the GITHUB_TOKEN to allow post comment with Preview URL
+    permissions:
+      issues: write
```

Comment:
Is `issues: write` required as well?
The action will only comment on PRs, right?